### PR TITLE
Fix bad application template paths

### PR DESCRIPTION
**Pull request by FredKSchott**
Change absolute URL paths to relative ones so that build is happy & doesn't fail.

/cc @justinfagnani

### CODE REVIEW

**Comment by abdonrd** [comment]
I remember @justinfagnani said the `index.html` should have absolute paths.

**Comment by FredKSchott** [comment]
Ah, I hadn't thought of that. That makes sense (and works) for the app-shell model (where shell, entrypoint, etc are all defined in a polymer.json). Is that also true for an older monolith-style application?

File: src/init/application/templates/index.html [hunk 1]
```diff
@@ -6,8 +6,8 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="manifest" href="/manifest.json">
-    <script src="/bower_components/webcomponentsjs/webcomponents-lite.js"></script>
-    <link rel="import" href="/src/<%= elementName %>/<%= elementName %>.html">
+    <script src="bower_components/webcomponentsjs/webcomponents-lite.js"></script>
```
Comment (justinfagnani) [inline src/init/application/templates/index.html]: These really should be absolute though. In a routable app, they need to load from the root.

Comment (FredKSchott) [inline src/init/application/templates/index.html]: you're right, this isn't a solution. Closing to fix the problem in polymer-build instead!

**Comment by web-devel** [comment]
@justinfagnani what if I want to serve an app from some context path i.e:
http://somedomain/app/...

**Comment by FredKSchott** [comment]
@web-devel see https://github.com/Polymer/polymer-cli/issues/273

**Pull request closed without merging**